src/backend/distributed/sql/downgrades/citus--11.0-1--10.2-4.sql

```diff
@@ -94,7 +94,7 @@ ALTER TABLE citus.pg_dist_object DROP COLUMN force_delegation;
 SET search_path = 'pg_catalog';
 
 
-DROP FUNCTION IF EXISTS get_all_active_transactions();
+DROP FUNCTION IF EXISTS get_all_active_transactions() CASCADE;
```

Comment:
CASCADE seems scary. If a user somehow created a dependency between a table and this function, then we will inadvertently drop this table. Why do we need this?